.circleci/config.yml

```diff
@@ -13,7 +13,8 @@ jobs:
           keys:
           - build-dependencies-{{ checksum "build.sbt" }}
           - sbt-dependencies--{{ checksum "project/build.properties" }}
-      - run: sbt clean coverage +test
+      # Resource of Docker Executor - 2 vCPUs and 4GB
```

Comment:
can we add this: `SBT_OPTS: "-XX:+CMSClassUnloadingEnabled -XX:PermSize=256M -XX:MaxPermSize=512M -Xmx2G"`?

Reply:
That's great! Makes sense!